Gopkg.toml

```diff
@@ -40,14 +44,19 @@
   name = "google.golang.org/grpc"
   version = "1.9.2"
 
-[[constraint]]
+[[override]]
```

Comment:
these probably do not have to be overrides, constraint should be better for most of these as they will behave correct if dependencies are removed/changed. We only need to override if we transitively depend on a certain version of a package, but I believe most of these are direct dependencies so we should just use `[[constraint]]`

Reply:
I added "override" because I ran into errors earlier without them.  I just removed them and re-ran them.  It works now.